sources/packages/api/src/services/application/application.service.ts

```diff
@@ -343,17 +343,17 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "institutionType",
         "student",
       ])
-      .innerJoin("application.programYear", "programYear")
-      .innerJoin("application.offering", "offering")
-      .innerJoin("application.pirProgram", "pirProgram")
-      .innerJoin("application.location", "location")
+      .leftJoin("application.programYear", "programYear")
```

Comment:
Program year should be an inner join. An application should not even start the assessment without it.